qt/TorrentDelegate.cc

```diff
@@ -578,6 +584,27 @@ void TorrentDelegate::drawTorrent(QPainter* painter, QStyleOptionViewItem const&
 
     painter->setFont(layout.name_font);
     painter->drawText(layout.name_rect, Qt::AlignLeft | Qt::AlignVCenter, layout.nameText());
+    painter->setFont(layout.labels_font);
+
+    // labels
+    int width_used = 0;
+    for (auto label_text : tor.labels())
+    {
+        QRect label_rect = QFontMetrics{ layout.labels_font }.boundingRect(label_text);
+        label_rect.moveTopLeft(layout.labels_rect.topLeft() + QPoint{ width_used, 0 });
+        width_used += label_rect.width();
+
+        // gap between labels
+        width_used += 5;
+
+        auto saved_brush = painter->brush();
+        painter->setBrush(QBrush{ "lightgrey" });
```

Comment:
_(nit)_ might be worth making a reusable QBrush to avoid rebuilding this every time? e.g. as a sibling of the `QColor const` fields around TorrentDelegate.h  line 50

Reply:
done - using existing colors instead of defining new ones now